src/zenml/enums.py

```diff
@@ -73,3 +73,15 @@ class StoreType(StrEnum):
     LOCAL = "local"
     SQL = "sql"
     REST = "rest"
+
+
+class ContainerRegistry(StrEnum):
```

Comment:
I think in all other cases we handle these flavors as simple string constants now, but I can see the point of having them as an enum with that many zenml core implementations. I'd still vote for changing the enum name to `ContainerRegistryFlavor` or similar